### DISCUSSION TITLE
[New Feature Request] Video Long Description

### DISCUSSION THREAD

**Discussion by AndrewRCD** [rrirower/Channel-Feed-Manager#158] [Feature requests]
I noticed that the long video description is not added to the search feed.

Maybe you can look into it and add that in the next update. It would help a lot in ranking :) and search.

Thanks

**Comment by rrirower** [comment]
I've run a test on some sample feeds and the Search feed does contain the long description.  It is added to the Search feed if it is not empty and it is different from the short description.  If you do not see the long description in your Search feed, I'll need your JSON feed file to test.

**Comment by rrirower** [comment]
No action.
